src/jimgw/single_event/likelihood.py

```
@@ -683,17 +722,21 @@
     h_sky: dict[str, Float[Array, " n_dim"]],
     detectors: list[Detector],
     freqs: Float[Array, " n_dim"],
+    datas: list[Float[Array, " n_dim"]],
+    psds: list[Float[Array, " n_dim"]],
     align_time: Float,
     **kwargs,
 ) -> Float:
     log_likelihood = 0.0
     df = freqs[1] - freqs[0]
+    f_min = freqs[0]
```

Ruff (0.8.2)

Local variable `f_min` is assigned to but never used

Remove assignment to unused variable `f_min`

(F841)